### ISSUE TITLE
Automatic Btrfs - Boot Partition wrong

### ISSUE THREAD

**Issue by gwutz** [deadhead420/arch-linux-anywhere#354]
Hey i love that script. But i tried it in a virtual machine with automatich LVM and encryption and choose Btrfs as my filesystem. He wrote for the boot partition btrfs into fstab which was wrong (should be ext4) so the first start of the machine failed.

**Comment by pnedkov** [comment]
#365 

This should fix it.
When `Auto Partition Drive` or `Auto partition encrypted LVM` option is selected the file system of the boot partition is hardcoded to ext4.

**Comment by pnedkov** [comment]
@gwutz did you get a chance to test it? Please close the issue if it works for you. Thanks!

**Comment by deadhead420** [comment]
Thanks @pnedkov 
I've invited you to be a collaborator

**Comment by pnedkov** [comment]
Thanks @deadhead420 .  Appreciate it. Where have you been? Vacation or just life happened?

**Comment by deadhead420** [comment]
@pnedkov Just life. Have been really busy with work and stuff. Sometimes I lose interest in coding and tech stuff for a bit to do other things but I always come back. I finally have a bit of time so here I am :)

I fixed some major bugs in fetchmirrors and fetchpkg and did a release on the AUR. I'm working on a new arch anywhere xfce desktop now and getting ready to release 2.2.9. I'm cleaning up the default desktop and making it more "default" looking out of the box, mainly changing the wallpaper but I've also made a few other changes. I also want to add an option for "AAXFCE Extras" which will install a more full and user ready desktop out of the box (will include firefox, gimp, office suite, media software).
